config/runtime.exs

```diff
@@ -11,7 +11,8 @@ config :elixir_boilerplate,
 config :elixir_boilerplate, ElixirBoilerplate.Repo,
   url: get_env!("DATABASE_URL"),
   ssl: get_env("DATABASE_SSL", :boolean),
-  pool_size: get_env!("DATABASE_POOL_SIZE", :integer)
+  pool_size: get_env!("DATABASE_POOL_SIZE", :integer),
+  socket_options: if(get_env("ECTO_IPV6", :boolean), do: [:inet6], else: [])
```

Comment:
Souldn't we use a name aligned with other `DATABASE_` environment variables?

Something like `DATABASE_IPV6` might make more sense along side the multiple ENV we already have.

Reply:
There are not many `ENV` in a vanilla `phx.new` project and we added a few important pieces to get our baseline as _12-Factor_ as possible.

We might want to add it in `.env.dev` and `.env.test` (even if there’s a smart default in `ElixirBoilerplate.Config.get_env/2` ) for documentation/discoverability.